### ISSUE TITLE
Knockdown / Multi-target bug

### ISSUE THREAD

**Issue by SamWilby** [openlegend/core-rules#419]
Just had some confusion that caught a bug in the current rules. 

As written, Knockdown gives advantage 1 to melee and area attacks against the target and disadvantage 1 for ranged attacks. This is flavorful and easy to remember, but doesn't work when the prone unit is part of a multi-target attack.

For example:  you aim an Area (Cube) attack and catch 2 enemies in a 10' Cube, one is prone, the other isn't; what dice do you roll? How can you have advantage against 1 of them but not the other when you're making a single roll?

Suggested fix: characters afflicted by Knockdown get +1 to Guard versus Ranged attacks and -1 Guard versus Melee attacks (similar to how Invisible is worded for the same reason). Area could also be added, but simplicity and cutting down on word count I think it makes sense that Area attacks would be unaffected. This also removes the problem with effects that target Resolve gaining advantage because the target is prone.

@istabosz at @brianfeister's request.

**Comment by istabosz** [comment]
Makes sense to me.

@brianfeister Do you recall the statistical bonus that advantage provides? Would a +2/-2 modifier be more appropriate?

**Comment by ucffool** [comment]
I ran it through the new dice stat generator and advantage never gives more than a +1 over *none* in expected value, so +1/-1 seems appropriate.

**Comment by ucffool** [comment]
As an aside, I ran into this as well running a game this weekend and handled it by taking the most neutral option (standing and prone target, no advantage). I just tried writing a rule for it and it was just confusing and didn't make sense. @SamWilby suggestion of moving the burden onto the target makes more sense, but that concept burdens the system in other places.

For instance (this happened this weekend as well), if you are flanking an enemy with two friends and the GM gives you ADV1 on your attack for ganging up, but you multi-target another enemy that  is only adjacent to you... same problem. This starts breaking down the foundations of OL and adv/dis system. 

tl;dr **Can of Worms**

**Comment by istabosz** [comment]
This isn't the cleanest of rules, but I imagine it wouldn't occur frequently enough to bog the game down that much. And it does solve the issue:

"If you are multi-targeting and would apply different levels of advantage or disadvantage to different targets, make a separate action roll for each group of targets. For purposes of the _every roll matters_ principle, the action counts as a success if _any_ of these rolls succeed.

**Comment by oconnor0** [comment]
@ucffool The dice stat generator at https://openlegend.github.io/open-legend-stat-gen/index.html is incorrect. Advantage 0 vs Advantage 1 ranges from +3.85 at Attribute 0, to +1 at Attribute 1 to +3.5 at Attribute 9.

http://anydice.com/program/ba52

**Comment by brianfeister** [comment]
@oconnor0 is correct regarding numeric average adjustment. Advantage / Disadvantage is +2 / -2 (closer to 2.5 if being precise). So the answer to the earlier question is +2 / -2 as a flat numeric modifier. 

I think @SamWilby's suggestion is by far the cleanest solution proposed thus far:

> Characters afflicted by Knockdown get +1 to Guard versus Ranged attacks and -1 Guard versus Melee attacks.

**Comment by istabosz** [comment]
Just updated this fix in the print edition as well in 03c9a86
